packages/strapi-design-system/src/Flex/FlexProps.js

```diff
@@ -1,43 +0,0 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-
-export const FlexProps = (props) => <div {...props} />;
-
-export const flexDefaultProps = {
-  alignItems: 'center',
-  basis: undefined,
-  direction: 'row',
-  gap: undefined,
-  inline: false,
-  justifyContent: undefined,
-  reverse: false,
-  shrink: undefined,
-  wrap: undefined,
-};
-
-export const flexPropTypes = {
-  alignItems: PropTypes.string,
-  basis: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  direction: PropTypes.string,
-  /**
-   * Supports responsive values
-   */
```

Comment:
These comments are useful for [ArgsTable](https://storybook.js.org/docs/react/writing-docs/doc-block-argstable). Can you make sure that the documentation is still displayed, please?

![Screenshot 2022-12-05 at 13 17 03](https://user-images.githubusercontent.com/7756284/205635289-8b06d4d0-448d-4af9-9fd2-189812a483c6.png)